# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gn/version.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Prevent the build from trying to generate a version number using a .git dir
--- gn-69ec4fc/build/gen.py.orig	2021-10-24 19:44:02.604535132 +0200
+++ gn-69ec4fc/build/gen.py	2021-10-24 19:46:04.759540562 +0200
@@ -157,9 +157,9 @@
   out_dir = options.out_path or os.path.join(REPO_ROOT, 'out')
   if not os.path.isdir(out_dir):
     os.makedirs(out_dir)
-  if not options.no_last_commit_position:
-    GenerateLastCommitPosition(host,
-                               os.path.join(out_dir, 'last_commit_position.h'))
+  #if not options.no_last_commit_position:
+  #  GenerateLastCommitPosition(host,
+  #                             os.path.join(out_dir, 'last_commit_position.h'))
   WriteGNNinja(os.path.join(out_dir, 'build.ninja'), platform, host, options)
   return 0
 
